data-raw/depth/depths.R

```diff
@@ -1,6 +1,6 @@
 
 depths = function(){
-
+	library(dplyr)
```

Comment:
Is this necessary if in `remake.yml` already?

Reply:
No, this is redundant.